src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService checkByEmail = new UserService();
+        if (checkByEmail.findByEmail(email) != null) {
+            User newUser = checkByEmail.findByEmail(email);
```

Comment:
Should we search the user inside the database twice?

Reply:
Resolved